css/style.css

```diff
@@ -192,15 +177,107 @@ h2 {
 
 .due-date {
   display: inline-block;
-  margin-bottom: 0.5rem;
+  margin-bottom: .5rem;
 }
 
+
 /* ----------------- footer ----------------- */
 
 .primary-footer {
-  width: 100vw;
   font-size: 1.2rem;
   text-align: center;
-
+  
   padding: 1rem 0;
 }
+/* ------- media queries */
```

Comment:
Looks like these could be a bit dryer.  We might want to do mobile first design going forward like Leon mentions and then scale up. Also I believe rem scales automagically with the size of viewport, so I'm not sure if we need all these minor changes for each media query.  Might need to hop on a call and understand better what you're seeing.

Typical breakpoints example from bootstrap:
  xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px

Reply:
I agree, this isn’t complete or anything. I just wanted to clean up what was there before and make it a little better, not make it perfect. But if you have ideas please feel free to edit the code. This is just to get the ball rolling with media queries